addons/html_builder/static/src/website_builder/website_mass_mailing/mailing_list_subscribe_option.js

```diff
@@ -0,0 +1,19 @@
+import { Component, useState, onWillStart } from "@odoo/owl";
+import { defaultBuilderComponents } from "../../core/default_builder_components";
+
+export class MailingListSubscribeOption extends Component {
+    static template = "html_builder.MailingListSubscribeOption";
+    static components = { ...defaultBuilderComponents };
+    static props = {
+        fetchMailingLists: Function,
+    };
+
+    setup() {
+        this.state = useState({
+            mailingLists: [],
+        });
```

Comment:
I think here we do not have to use the state, we can directly use `this.mailingLists`.